Plugins/SaveToRedux/SaveToRedux.plugin.js

```diff
@@ -873,6 +874,9 @@ module.exports = (() => {
         const { extension, previewDate = null, previewRand = null, throwFail = false, onlyDir = false } = options;
         const date = previewDate || new Date();
         const rand = previewRand || this.rand();
+
+        //if filename is nothing but 'SPOILER_' default to 'unknown'
+        name = this.settings.saveOptions.removeSpoilerTag && name.startsWith("SPOILER_") ? ((name = name.substr(8)) ? name : "unknown") : name;
```

Comment:
This is not optimal, prefer doing it as
```
if (this.settings.saveOptions.removeSpoilerTag && !name.indexOf('SPOILER_')) name = name.substr(8) || 'unknown';
```

Reply:
Aight, let me figure out how to edit it, first time editing a PR lmfao.

Reply:
Ok, rebased and force pushed so there isn't an extra unnecessary commit. This way should probably work(?)